assignment/scripts/loops-practice.js

```diff
@@ -13,18 +13,27 @@ for (let i=0; i<4; i++) {
 // 1.a. TODO: Write a for loop to console.log the numbers from 0 to 5 
 //   - Which part of the example loop do you need to change to do this?
 console.log('count from 0 to 5');
-
+for (let i = 0; i < 6; i++) {
+  console.log(i);
+}
 // 1.b. TODO: Write a for loop to console.log the numbers from 3 to 5
 //   - Which part of the example loop do you need to change to do this?
 console.log('count from 3 to 5');
-
+for (let i = 3; i < 6; i++) {
+  console.log(i);
+}
 // 1.c. TODO: Write a for loop to console.log EVEN numbers from 2 to 10
 //   - Which part of the example loop do you need to change to do this?
 console.log('count even numbers from 2 to 10 (2, 4, 6, 8, 10):');
+for (let i = 2; i < 11; (i += 2)) {
+  console.log(i);
+} 
 
 // 1.d. (STRETCH) TODO: Write a for loop to do a countdown from 5 to 0
 console.log('STRETCH: countdown from 5 to 0');
-
+for (let i = 5; i > -1; i--) {
```

Comment:
Although it would work the same, I might have used `i >= 0` here just because it makes the _intent_ more clear. I don't care about -1. I care about 0.